browser/extensions/brave_component_loader.cc

```diff
@@ -108,4 +111,12 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
   }
 }
 
+void BraveComponentLoader::AddEthereumRemoteClientExtension() {
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
not sure if this buildflag is necessary because it will never get called unless `BRAVE_WALLET_ENABLED`

Reply:
if anything I would probably wrap both `AddEthereumRemoteClientExtension` and `RemovEthereumRemoteClientExtension` entire methods in `BRAVE_WALLET_ENABLED` to ensure that

Reply:
removed `RemovEthereumRemoteClientExtension` and put the flag to be around the whole member function declaration and definition.